packages/config/config/devices/0x015f/mh-s411-7102.json

```diff
@@ -0,0 +1,231 @@
+					"value": 1
+				},
+				{
+					"label": "Click to turn off, auto on after param 6 duration",
+					"value": 2
+				},
+				{
+					"label": "Hold >3s for on until released",
+					"value": 3
+				},
+				{
+					"label": "Single click to toggle, hold >3s for on until released",
+					"value": 4
+				}
+			]
+		},
+		{
+			"#": "6",
+			"label": "Relay On Duration",
+			"description": "0 for infinite",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "8",
+			"label": "Basic CC Mode",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Relay 1 responds to set, relay 1 responds to get, instant updates disabled",
+					"value": 0
+				},
+				{
+					"label": "Relay 1 responds to set, relay 1 responds to get, instant updates enabled",
+					"value": 1
+				},
+				{
+					"label": "All relays respond to set, get ignored, instant updates disabled",
+					"value": 2
+				},
+				{
+					"label": "All relays respond to set, relay 1 responds to get, instant updates disabled",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Respond to Scenes",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Scene Activate Mode",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Scene activate function disabled",
+					"value": 0
+				},
+				{
+					"label": "Button 1 tap activates scene in param 12 regardless of state",
+					"value": 1
+				},
+				{
+					"label": "Button 1 tap activates scene in param 12 when state is off",
+					"value": 2
+				},
+				{
+					"label": "Button 1 tap activates scene in param 12 when state is on",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "12",
+			"label": "Button 1 Activate Scene ID",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 255,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		{
+			"#": "13",
+			"label": "Button 1 Activate Scene Dim Duration",
+			"description": "0-127 seconds, +128 for minutes",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "255",
+			"label": "Factory Reset",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal Operation",
+					"value": 0
+				},
+				{
+					"label": "Reset",
+					"value": 85
+				}
+			]
+		}
+	],
+	"compat": {
+		"commandClasses": {
+			"remove": {
+				"Scene Actuator Configuration": {
+					"endpoints": "*"
+				},
+				"Battery": {
```

Comment:
Same question. According to your logfile, this is a FLiRS device with a battery. Or is that incorrect?
```
is always listening:   false
is frequent listening: 250ms
```

Reply:
This isn't a battery powered devices, I don't know why it is reconized as this. I removed those endpoint because they are useless aswell. If you know a way to make this product type reconnized correctly, I would happily test it

Reply:
> I don't know why it is reconized as this

Because the device says so.

> If you know a way to make this product type reconnized correctly

The manufacturer would have to fix their firmware for this.

We can remove the battery support here, but as far as the mesh is concerned, the device will be treated as a frequently-listening battery-powered device. Meaning you can send it commands as usual without having to wake it, but it won't act as a repeater for other devices.

Reply:
Well, I'll tell it to the manufacturer. I keep you updated